interface/super/rules/controllers/edit/helper/common.php

```diff
@@ -66,17 +66,29 @@
 <!-- -->
 <!-- -->
 <!-- -->
+
+<!-- original version:
```

Comment:
git will let other developers see/compare code lines from prior versions. So letting script reflect only the changes makes it easier to read/maintain.  Only exception could be when you expect someone may want to include original code for specific (non-standard) use cases.

Reply:
so it's usually okay @hanksterr7 to remove the code you're replacing but leave your insight in these comments